app/pb_kits/playbook/pb_date/_date.jsx

```diff
@@ -28,9 +29,10 @@ const PbDate = (props: PbDateProps) => {
     id,
     showDayOfWeek = false,
     showIcon = false,
+    timeZone,
   } = props
 
-  const dateTimestamp = new DateTime({ value: date })
+  const dateTimestamp = new DateTime({ value: date, zone: timeZone })
```

Comment:
@jasoncypret is this the spot you mentioned about multiple timezones being included?

cc:@coleerikson

Reply:
I think the one @jasoncypret mentioned is here: https://github.com/powerhome/playbook/pull/1080